notebooks/official/pytorch/pytorch-text-sentiment-classification-custom-train-deploy.ipynb

```diff
@@ -0,0 +1,1994 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %%bash -s $APP_NAME
</pre><p>What does this do? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
Here in the cell, a bash command is being run by passing the "APP_NAME" variable as an argument.